docs/developer-center/examples/basic-usage.md

```diff
@@ -0,0 +1,604 @@
+
```

Comment:
This file is out of date and has been replaced/renamed: https://github.com/CartoDB/cartoframes/blob/master/docs/developer-center/examples/BasicUsage.md